terraform/aks/database.tf

```diff
@@ -0,0 +1,31 @@
+module "postgres" {
+  source = "./vendor/modules/aks//aks/postgres"
+
+  namespace                   = var.namespace
+  environment                 = local.environment
+  azure_resource_prefix       = var.azure_resource_prefix
+  service_name                = var.service_name
+  service_short               = var.service_short
+  config_short                = var.config_short
+  cluster_configuration_map   = module.cluster_data.configuration_map
+  use_azure                   = var.deploy_azure_backing_services
+  azure_enable_monitoring     = var.enable_monitoring
+  azure_enable_backup_storage = var.enable_postgres_backup_storage
+  server_version              = "14"
+}
+
+
+module "redis-cache" {
```

Comment:
Can we pin server_version to 6?

Reply:
default is 6 in the module?
https://github.com/DFE-Digital/terraform-modules/blob/main/aks/redis/variables.tf#L49

Reply:
Yes but we may want to default to 7 in the module once it's available. So it should be pinned in GIT so GIT it is not updated automatically.

Reply:
pinned to 6